mozilla_django_oidc_db/fields.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from django_jsonform.models.fields import ArrayField
+
+
+class ClaimField(ArrayField):
```

Comment:
Correct, and as a way to avoid repeating the nested child definition so that if we need to tweak that, there's a single place in the code base to change :)